coderedcms/project_template/project_name/settings/base.py

```diff
@@ -29,6 +29,7 @@
 
     # CodeRed CMS
     'coderedcms',
+    'coderedcms.wagtail_flexible_forms',
```

Comment:
Is this really needed here? The code is already part of coderedcms. Unless there is some kind of wagtail magic where it needs to be its own app.

Reply:
As the code has their own models and hooks, it needs to be defined here.  The model issue will be rectified if we treat wff as a library and extend everything in it.  But we might need to repeat the code in their hooks.